<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>361</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>597</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Web Search</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="parametersGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>351</width>
          <height>145</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>351</width>
          <height>145</height>
         </size>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>331</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="urlLabel">
            <property name="minimumSize">
             <size>
              <width>23</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>23</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>URL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="urlEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>53</y>
           <width>331</width>
           <height>23</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="findLabel">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Find:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="findEdit">
            <property name="minimumSize">
             <size>
              <width>299</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>299</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>82</y>
           <width>331</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="threadsLabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>73</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>Max threads:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="threadSpinBox">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>112</y>
           <width>331</width>
           <height>23</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="maxUrlslabel">
            <property name="minimumSize">
             <size>
              <width>73</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>73</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string>Max URLs:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="maxUrlslSpinBox">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>95</width>
          <height>145</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>95</width>
          <height>145</height>
         </size>
        </property>
        <property name="title">
         <string>Buttons</string>
        </property>
        <widget class="QPushButton" name="startButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>23</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>52</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>579</width>
        <height>192</height>
       </size>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
